libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Prettier tries to change `Boolean` to `boolean`

Reply:
Arf, yeah, Boolean is also probably understood by Typescript internally, so it probably can't find our type defintion